src/main/php/PDepend/Metrics/Analyzer/CodeRankAnalyzer.php

```diff
@@ -113,7 +113,7 @@ class CodeRankAnalyzer extends AbstractAnalyzer implements AnalyzerNodeAware
      *
      * @var array<string, array<string, int>>
      */
-    private $nodeMetrics = null;
+    private $nodeMetrics;
```

Comment:
Better to keep the default value and switch to typed property.
```suggestion
    private ?array $nodeMetrics = null;
```

Reply:
Hm I was thinking that most of these where set as a form of "invalid" or "Uninitialized" state.

To keep this PR simple, by just have rule based changes in it, I have removed the rule that causes this change. We can then revisit it individually at a later point.

Reply:
Here we go :)

https://github.com/pdepend/pdepend/pull/806